runtime/compiler/control/CompilationThread.cpp

```diff
@@ -7070,6 +7070,12 @@ TR::CompilationInfoPerThreadBase::wrappedCompile(J9PortLibrary *portLib, void *
                options->setOption(TR_EnableAnnotations,false);
             }
 
+         if (vm->canUseSymbolValidationManager() && options->getOption(TR_EnableSymbolValidationManager))
+            {
```

Comment:
Now I realize that to control this feature externally one has to use the TR_EnableSymbolValidationManager option. This makes TR_UseSymbolValidationManager truly a flag and not an option. Maybe we should put this into flags32_t _flags; in OMRCompilation.